blockchain/error.go

```diff
@@ -316,52 +316,62 @@ const (
 	// than were allowed.
 	ErrTooManyRevocations
 
-	// ErrSStxCommitment indicates that the propotional amounts from the inputs
-	// of an SStx did not match those found in the commitment outputs.
-	ErrSStxCommitment
+	// ErrTicketCommitment indicates that a ticket commitment contains an amount
+	// that does not coincide with the associated ticket input amount.
+	ErrTicketCommitment
 
-	// ErrInvalidSSGenInput indicates that the input SStx to the SSGen tx was
-	// invalid because it was not an SStx.
-	ErrInvalidSSGenInput
+	// ErrInvalidVoteInput indicates that an input to a vote transaction is
+	// either not a stake ticket submission or is not a supported version.
+	ErrInvalidVoteInput
 
-	// ErrSSGenPayeeNum indicates that the number of payees from the referenced
-	// SSGen's SStx was not the same as the number of the payees in the outputs
-	// of the SSGen tx.
-	ErrSSGenPayeeNum
+	// ErrBadNumPayees indicates that either a vote or revocation transaction
+	// does not make the correct number of payments per the associated ticket
+	// commitments.
+	ErrBadNumPayees
 
-	// ErrSSGenPayeeOuts indicates that the SSGen payee outputs were either not
-	// the values that would be expected given the rewards and input amounts of
-	// the original SStx, or that the SSGen addresses did not correctly correspond
-	// to the null data outputs given in the originating SStx.
-	ErrSSGenPayeeOuts
+	// ErrBadPayeeScriptVersion indicates that either a vote or revocation
+	// transaction output that corresponds to a ticket commitment does not have
+	// use a supported script version.
```

Comment:
`... does not have use a supported script version.` should be `... does not use a supported script version.`